test/validateRules.js

```diff
@@ -88,22 +88,5 @@ const ajv = new Ajv({ loadSchema, allErrors: true });
     exitWithError("Found duplicate rules");
   }
 
-  // 4. Check for empty rules that have no click or cookie injection rule.
```

Comment:
I wonder if we should only allow rules like this with a specific ID, such as `disabled` that you are using above.

Reply:
Oh... good idea! Let's do that instead.